composer.json

```diff
@@ -46,7 +46,7 @@
         "liip/monitor-bundle": "^2.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
         "swiftmailer/swiftmailer": "^4.3 || ^5.0",
-        "friendsofsymfony/rest-bundle": "^1.1",
+        "friendsofsymfony/rest-bundle": "^1.1 || ^2.1",
```

Comment:
To fix the failing job, you may bump this to the version that introduces the `addParam` method you need.